### PR TITLE
Feat/DTCORE-2010

### PR DESCRIPTION
**Pull request by seven-amid**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master` / `release/**` / `develop` 
| Bug fix?         | yes/no
| New feature?     | yes/no
| Breaking change? | yes/no
| Tickets          | Fix #... 
| License          | BSD 3-Clause

- [ ] Try to keep pull requests small so they can be easily reviewed.
- [ ] Commits are signed-off
- [ ] Only FR translations have been updated
- [ ] Branch is up-to-date with target branch
- [ ] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [ ] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/pci-public-ip/src/data/floating-ip.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        );
+
+        floatingIPResult.associatedEntity.name = instance ? instance.name : '';
+      }
+
+      return { ...floatingIPResult };
+    });
+
+    return aggregatedData;
+  } catch (error) {
```
Comment (frenautvh) [inline packages/manager/apps/pci-public-ip/src/data/floating-ip.ts:53]: you can remove the try/catch console.log

File: packages/manager/apps/pci-public-ip/src/components/list/FloatingIP.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  const { t } = useTranslation('common');
+
+  const { pagination, setPagination } = useDatagridSearchParams();
+
+  const { error, data: floatingIPs, isLoading } = useFLoatingIPs(
+    projectId || '',
+    { pagination },
+  );
+
+  const goToInstanceHref = (id: string) => `${projectUrl}/instances/${id}`;
```
Comment (frenautvh) [inline packages/manager/apps/pci-public-ip/src/components/list/FloatingIP.tsx:37]: for url pointing to other apps you can use navigation plugin from the shell to create URLs (passing application id: 'public-cloud')

Comment (seven-amid) [reply packages/manager/apps/pci-public-ip/src/components/list/FloatingIP.tsx:37]: the projectUrl is coming from the shell actually, it's declared in the parent component ListingPage
```
  useEffect(() => {
    navigation
      .getURL('public-cloud', `#/pci/projects/${projectId}`, {})
      .then((data) => {
        setProjectUrl(data as string);
      });
  }, [projectId, navigation]);
```

Or maybe i didn't understood well your suggestion ?

File: packages/manager/apps/pci-public-ip/src/data/floating-ip.ts (hunk 1 above)
Comment (seven-amid) [reply packages/manager/apps/pci-public-ip/src/data/floating-ip.ts:53]: Done

**Review by frenautvh — APPROVED** [review]

File: packages/manager/apps/pci-public-ip/src/components/list/FloatingIP.tsx (hunk 2 above)
Comment (frenautvh) [reply packages/manager/apps/pci-public-ip/src/components/list/FloatingIP.tsx:37]: ok if you are passing the URL this way it should be good

**Review by kqesar — APPROVED** [review]

**Pull request merged**